test/integration/short/client-execute-prepared-tests.js

```diff
@@ -12,8 +12,117 @@ var vit = helper.vit;
 var vdescribe = helper.vdescribe;
 var Uuid = types.Uuid;
 
+var SimulacronCluster = require('../simulacron-helper');
+
 describe('Client', function () {
   this.timeout(120000);
+  describe('Preparing statements on nodes behavior', function () {
+    var sCluster = null;
+    var client = null;
+    var query = util.format('SELECT * FROM ks.table1 WHERE id1 = ?');
+    beforeEach(function (done) {
+      utils.series(
+        [
+          function stopAnyCcmCluster(next) {
+            helper.ccmHelper.removeIfAny(next);
+          },
+          function startCluster(next) {
+            sCluster = new SimulacronCluster("5", "3.10", "", "test", true, 1, function (err) {
+              var clientOptions = utils.extend({}, helper.baseOptions);
+              clientOptions.contactPoints = [];
+              clientOptions.contactPoints.push(sCluster.dcs[0].nodes[0].address);
```

Comment:
You can expose a property or method on `SimulacronCluster` named `contactPoints` and use it to construct the client options:

```js
var client = Client({ contactPoints: sCluster.contactPoints() });
```